sympy/galgebra/printing.py

```diff
@@ -30,10 +30,14 @@
 
 import os
 import sys
+
```

Comment:
OK, I was guessing that. I am not sure if there are some PEP8 guidelines for such imports.